plugins/lookup/tss.py

```diff
@@ -277,8 +288,11 @@ def run(self, terms, variables, **kwargs):
             api_path_uri=self.get_option("api_path_uri"),
             token_path_uri=self.get_option("token_path_uri"),
         )
+        query_params = {}
+        if self.get_option("comment"):
+            query_params = {"autocomment": self.get_option("comment")}
```

Comment:
I would rewrite this as
```suggestion
            query_params["autocomment"] = self.get_option("comment")
```
(if you - or someone else - eventually want to add more query parameters).